src/Missle.hpp

```diff
@@ -4,16 +4,18 @@
 #include "MovementState.hpp"
 class Missle {
  public:
-  Missle(sf::Texture& texture, const MovementState& state);
+  Missle(sf::Texture& texture, const MovementState& state, unsigned int damage);
   void draw(sf::RenderWindow& window);
   void update();
   [[nodiscard]] sf::Vector2f get_pos() const;
+  [[nodiscard]] unsigned int get_damage() const;
 
   bool operator==(const Missle& rhs) const;
 
  private:
   sf::Sprite mSprite;
   float mSpeed = 20.0f;
+  unsigned int mDamage;
```

Comment:
I think it would be better to initialize this value here.

Reply:
I added the default value, not sure if I should remove `damage` parameter from c-tor?

Reply:
If it's used in tests - you don't have to.